turbinia/processors/google_cloud.py

```diff
@@ -25,13 +25,18 @@
 from six.moves import urllib
 
 from libcloudforensics.providers.gcp.internal import project as gcp_project
+from prometheus_client import Gauge
 from turbinia import config
 from turbinia import TurbiniaException
 
 log = logging.getLogger('turbinia')
 
 RETRY_MAX = 10
 
+turbinia_nonexisting_disk_path = Gauge(
+    'turbinia_nonexisting__disk_path',
```

Comment:
Looks like an extra underscore here?

```suggestion
    'turbinia_nonexisting_disk_path',
```